projects/aca-content/viewer/src/lib/components/viewer/viewer.component.scss

```diff
@@ -16,8 +16,12 @@
     }
   }
 
+  .adf-viewer-close-button {
+    padding-left: 12px;
+  }
+
   // todo: remove this when viewer supports extensions
-  .adf-viewer-toolbar > * > button:last-child {
+  .adf-viewer-toolbar > * > button:nth-last-child(3) {
```

Comment:
Hmm will this be working for every value of isCloseButtonOnLeft? 
Also especially after @MichalKinas comment here: https://github.com/Alfresco/alfresco-content-app/pull/3535/files#r1405984530 where button's position can be on left or right side.